README.md

```diff
@@ -853,6 +854,16 @@ Sometimes you have several work in progress commits that you want to combine bef
 
 This will do an interactive rebase that lists only the commits that you haven't already pushed, so it will be safe to reorder/fix/squash anything in the list.
 
+#### I need to abort the merge
```

Comment:
Needs the a link! See the other headings.

Reply:
Updated! thanks.